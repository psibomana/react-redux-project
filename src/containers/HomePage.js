import React from 'react';
import { Link } from 'react-router-dom';
import Header from '../components/common/Header';

const HomePage = () =>  {
  return (
    <div className="container">
      <Header/>
      <div className="jumbotron align">
        <h1>Pluralsight Administration</h1>
        <p>React, Redux and React Router in ES2015 for ultra-responsive web apps.</p>
        <Link to="about" className="btn btn-primary btn-lg">Learn more</Link>
      </div>
    </div>
  );
}

export default HomePage;
