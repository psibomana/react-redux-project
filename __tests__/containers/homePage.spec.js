import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomePage from '../../src/containers/homePage';

describe('The Home Page', () => {
  it('should render as expected', () => {
    const homePage = shallow(<HomePage />);
    expect(toJson(homePage)).toMatchSnapshot();
  })
});
