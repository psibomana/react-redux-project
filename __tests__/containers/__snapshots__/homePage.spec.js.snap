// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home Page should render as expected 1`] = `
<div
  className="container"
>
  <Header />
  <div
    className="jumbotron align"
  >
    <h1>
      Pluralsight Administration
    </h1>
    <p>
      React, Redux and React Router in ES2015 for ultra-responsive web apps.
    </p>
    <Link
      className="btn btn-primary btn-lg"
      replace={false}
      to="about"
    >
      Learn more
    </Link>
  </div>
</div>
`;
